snap/info.go

```diff
@@ -474,6 +514,11 @@ func (s *Info) Filename() string {
 	return filepath.Base(s.MountFile())
 }
 
+// ContRevision returns the snap revision.
+func (s *Info) ContRevision() Revision {
```

Comment:
same here

Reply:
This method has been removed now

Reply:
so the implementation should also be removed

Reply:
Oops... removed now